.github/workflows/deployment-template.yml

```diff
@@ -38,6 +38,8 @@ on:
         required: false
       kibanaserver:
         required: true
+      query_datasources_master_key:
```

Comment:
is this key will be changed regularly? do we need to keep update the key often?

Reply:
No..unless there is a leakage.